src/firebolt/async_db/_types.py

```diff
@@ -263,3 +258,23 @@ def process_token(token: Token) -> Token:
         )
 
     return formatted_sql
+
+
+def format_sql(query: str, parameters: Sequence[ParameterType]) -> str:
+    return format_statement(parse_sql(query)[0], parameters)
+
```

Comment:
Is this function used anywhere?

Reply:
Oh, no. Should remove it
It was used temporary